test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -76,6 +76,8 @@ protected BaseTestBaseline(ITestOutputHelper output) : base(output)
         protected StreamWriter LogWriter;
         private protected ConsoleEnvironment _env;
         protected IHostEnvironment Env => _env;
+        private IFileHandle _fileHandle;
```

Comment:
I wouldn't add this functionality to the base test class just for a single spot. Let's leave this off the base class, and just create and dispose the temp file in the one place that needs it.

Reply:
comment for next PR